### DISCUSSION TITLE
Change behavior of FWD/RWD to move by program instead of block

### DISCUSSION THREAD

**Discussion by factus10** [timex-sinclair-projects/TS-Pico#18] [General]
@ryangray idea to change from moving by program unit instead of by internal block.

**Comment by ryangray** [comment]
I think we also mentioned that the plain command `SAVE "tpi:fwd"` would move by file, and the `CODE m,n` option would allow you to move by block. We debated on exactly how though. 

One way, `m` could be number of files (2 blocks), and `n` be number of blocks. So it would move by 2*m+n blocks. You might typically use only one or the other:  `CODE 3,0` or `CODE 0,1`.

A choice might be made on the interpretation of `m` whether it is two blocks or if it moves header to header of the files. For example, if the tap pointer is on a data block, does `CODE 1,0` move one block to sync to the next header or always move two blocks? Maybe this *sync to next header* feature could be the special case of `CODE 0,0` instead.
